CIP-0140/README.lagda.md

```diff
@@ -0,0 +1,1568 @@
+| Algorand   |                     10 |                          1 |
+| Aptos      |                     50 |                          5 |
+| Avalance   |                     20 |                          1 |
+| Bitcoin    |                      3 |                         30 |
+| Cardano    |                     15 |                         10 |
+| Dogecoin   |                     40 |                         40 |
+| Ethereum   |                     70 |                         14 |
+| Polkadot   |                    n/a |                          5 |
+| Ripple     |                    n/a |                          0 |
+| Solana     |                    n/a |                          0 |
+| Tezos      |                      6 |                          3 |
+
+[^7]: Data extracted from https://support.kraken.com/hc/en-us/articles/203325283-Cryptocurrency-deposit-processing-times on 7 August 2024.
+
+
+### Feasible values for Peras protocol parameters
+
+Based on the analyses in the [Peras Technical Report #2](https://peras.cardano-scaling.org/docs/reports/tech-report-2#defining-protocol-parameters-values), a reasonable set of default [protocol parameters](#protocol-parameters) for further study, simulation, and discussion is show in the table below. The optimal values for a real-life blockchain would depend somewhat upon external requirements such as balancing settlement time against resisting adversarial behavior at high values of adversarial stake. This set of parameters is focused on the use case of knowing soon whether a block is settled or rolled back; other sets of parameters would be optimal for use cases that reduce the probability of roll-back at the expense of waiting longer for settlement.
+
+| Parameter              | Symbol           | Units   | Value | Rationale                                                            |
+| ---------------------- | ---------------- | ------- | ----: | -------------------------------------------------------------------- |
+| Round length           | $U$              | slots   |    90 | Settlement/non-settlement in under two minutes.                      |
+| Block-selection offset | $L$              | slots   |    30 | Several multiples of $\Delta$ to ensure block diffusion.             |
+| Certification boost    | $B$              | blocks  |    15 | Negligible probability to roll back boosted block.                   |
+| Security parameter     | $k_\text{peras}$ | blocks  |  3150 | Determined by the Praos security parameter and the boost.            |
+| Certificate expiration | $A$              | slots   | 27000 | Determined by the Praos security parameter and boost.                |
+| Chain-ignorance period | $R$              | rounds  |   300 | Determined by the Praos security parameter, round length, and boost. |
+| Cool-down period       | $K$              | rounds  |   780 | Determined by the Praos security parameter, round length and boost.  |
```

Comment:
How were these chosen? We have

 - $A = T_{\text{heal}} + T_{\text{CQ}}$, 
 - $R = \lceil A/U \rceil$ and 
 - $K = \lceil (A + T_{\text{CP}}) / U \rceil$. 

From the Praos parameterization (see e.g. [here](https://docs.google.com/document/d/1B8BNMx8jVWRjYiUBOaI3jfZ7dQNvNTSDODvT5iOuYCU/edit?tab=t.0#heading=h.qh2zcajmu6hm)), we have $T_{\text{CQ}} = k/f = 43200$ and $T_{\text{CP}} = 3k/f = 129600$, so we get

 - $A \ge 43200$,
 - $R \ge 480$,
 - $K \ge 1920$

for $U=90$. These values are already much larger than the ones listed here, and they have not yet considered $T_{\text{heal}}$.

Reply:
@corettis?

Reply:
@corettis?